neurolib/optimize/evolution/evolutionaryUtils.py

```diff
@@ -191,8 +184,12 @@ def printEvolutionInfo(evolution):
     print(f"Number of generations: {evolution.NGEN}")
```

Comment:
maybe all `print` -> `logging.info` ?

Reply:
This one is truly for printing, will be called only for `verbose=True`, otherwise it could be suppressed if the logger is configured differently...